Chrysalis-Papilio/Soen-490-Papilio-Android
# Papilio 77 contact

## General info
*Issue number*: PAPILIO-77

*Task description*:
- Display the email address of the companies on  their activity info page 
- make the email address a link to redirect to an email app on the phone

## Testing
*To test this feature you have to*:
1. Open the app
2. Login to your account 
3. click on an activity on the feed page 
4. scroll down to the bottom of the map where "Contact" field is displayed 
5. click on the link and verify that it redirect to an email app
6. after the redirection, verify that the "to" field is filled with the email address 
7. Verify also that the activities that are not from companies don't show the "Contact" field
